libpod/runtime_img.go

```diff
@@ -283,14 +283,12 @@ func (r *Runtime) LoadImage(ctx context.Context, name, inputFile string, writer
 			return dockerarchive.ParseReference(inputFile) // FIXME? We should add dockerarchive.NewReference()
 		},
 		func() (types.ImageReference, error) {
-			return ociarchive.NewReference(inputFile, name) // name may be ""
```

Comment:
- This is incorrect. OCI reference annotations have no semantics other than a set of allowed characters, and it’s perfectly possible fo the annotation in the image to have no slash. It must be possible to refer to such an image.
- `oci:` and `oci-archive:` should behave consistently; this changes the two in different ways.
- I can’t see any relationship to #7337 either way. One goal per PR, please; or at least one commit per concept; or _at the very least_ write down in the PR description what else the PR does!